### DISCUSSION TITLE
XLSX file signatures

### DISCUSSION THREAD

**Discussion by JamesPace-Carta** [jxlsteam/jxls#191] [Q&A] [answered]
Currently Apache POI is generating files with the Open XML Format magic bytes which leads to some issues uploaded files to certain websites. See https://stackoverflow.com/questions/55276423/upload-apache-poi-xlsx-generated-files-to-codeigniter-website

Is it possible to change or influence the magic bytes being generated by JXLS or Apache POI so the files can be detected correctly?

**Comment by SoltauFintel** [comment, marked as answer]
I think this is an Apache POI question.

It is important to use only Microsoft Excel.
